Fix bug in ndonnx.mean (and thus ndonnx.var and ndonnx.std) for negative axes (#124)
Quantco/ndonnx: CHANGELOG.rst

Changelog
=========

0.11.0 (2025-05-08)
-------------------

**Bug fix**

> - :func:`ndonnx.mean`, :func:`ndonnx.var`, and :func:`ndonnx.std` now produces correct result if negative axes are provided to the ``axis`` argument.

Author: ```suggestion
- :func:`ndonnx.mean`, :func:`ndonnx.var`, and :func:`ndonnx.std` now produce correct results if axes outside the range of `[-1, 1]` are given.
```
